app/models/translation_override.rb

```diff
@@ -50,13 +50,11 @@ class TranslationOverride < ActiveRecord::Base
   enum status: { up_to_date: 0, outdated: 1, invalid_interpolation_keys: 2, deprecated: 3 }
 
   scope :mf_locales,
-        ->(locale) do
-          where(locale: locale).where.not(status: "deprecated").where("translation_key LIKE '%_MF'")
-        end
+        ->(locale) { not_deprecated.where(locale: locale).where("translation_key LIKE '%_MF'") }
```

Comment:
As we’re using an `ActiveRecord` enum for the status, it gives us access to nice shortcuts like `not_<status>` :slightly_smiling_face:

Reply:
Wow, learned new grammar!